src/stories/EntryButton.stories.tsx

```diff
@@ -10,27 +11,27 @@ export default {
   component: EntryButton,
   argTypes: {
     type: {
-      options: ["Primary", "Secondary"],
+      options: ["Add", "Cancel"],
       control: { type: "select" },
     },
-    children: {
-      control: { type: "input" },
+    width: {
```

Comment:
If you have this property, even for the experiment, it's nice for the Crash Course to indicate that this is an experiment so that it's clear this is something outside of the regular scope. Usually notes are added like so:
`// NOTE: This is a note`
or
`// TODO: This and that needs to be updated`